make/common/native/Link.gmk

```diff
@@ -186,14 +186,17 @@ define CreateDynamicLibraryOrExecutable
 	      $$($1_STRIP) $$($1_STRIPFLAGS) $$($1_TARGET))
         endif
         ifeq ($$($1_CREATE_DEBUGINFO), true)
-          ifeq ($(call isTargetOs, linux), true)
+          ifeq ($(call isTargetOs, linux bsd), true)
             # Run this after strip is called, since strip can sometimes mangle
             # the embedded debuglink, which we want to avoid.
 	    $$(call ExecuteWithLog, $$($1_OBJECT_DIR)/$$($1_SAFE_NAME)_add_debuginfo_link, \
 	        $(CD) $$($1_SYMBOLS_DIR) && \
 	            $$($1_OBJCOPY) --add-gnu-debuglink=$$($1_DEBUGINFO_FILES) $$($1_TARGET))
           endif
         endif
+        ifeq ($(OPENJDK_TARGET_OS_ENV), bsd.netbsd)
+          /usr/sbin/paxctl +m $$@
```

Comment:
This will need to be turned into `$(PAXCTL) +m $$@`, with the paxctl tool checked in configure.

Reply:
Ack, will look into it.